.circleci/package/control/postinst

```diff
@@ -111,8 +111,8 @@ elif [[ -f /etc/debian_version ]]; then
     # Moving these lines out of this if statement would make `rmp -V` fail after installation.
     chown -R -L influxdb:influxdb $LOG_DIR
     chown -R -L influxdb:influxdb $DATA_DIR
-    chmod 755 $LOG_DIR
-    chmod 755 $DATA_DIR
+    chmod 750 $LOG_DIR
+    chmod 750 $DATA_DIR
```

Comment:
At some point, I'd like to change this so it respects the directory permissions set by the system administrator. (i.e they've either added `dpkg-statoverride` or manually executed `mkdir` then chown/chmod` themselves). However, since we've always been updating the permissions/ownership without complaint, I think we're good to move forward regardless!